index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Oumaima's Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo-placeholder">
+            <a href="#" class="logo">Ouma NJ</a>
+        </div>
+        <nav class="nav-links">
+            <ul class="menu-bar">
+                <li class="hamburger-button">
+                    <img src="./images/icon-Menu.svg" alt="Menu button" class="menu-button" id="hamburger"/>
+                </li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Weldone with the menu button :clap: but I think you still have to work on the alignment with your logo. A center-alignment will improve its current look, giving your users a more nice user experience with you impressively impeccable asthethics :smile: 
- Kindly consider the screenshot below and compare it with Figma design :point_up_2: to be sure how it looks:

![image](https://user-images.githubusercontent.com/99715640/219480285-705643f1-bcbb-4df5-86a2-57968e140d94.png)

- [ ] Also, while on a max-width of 375px, the menu button is not displaying fully. Kindly check your spacing (gap, padding, margin) to be sure what's off and try to fix it. This will increase a show of your dexterity with good designs. :muscle: Let's do this! :smile: